synapse/tests/test_lib_agenda.py

```diff
@@ -1097,3 +1113,119 @@ async def task():
                     self.eq(cdef01.get('lastresult'), 'cancelled')
                     self.gt(cdef00['laststarttime'], 0)
                     self.eq(cdef00['laststarttime'], cdef01['laststarttime'])
+
+    async def test_cron_creator_migration(self):
+        async with self.getRegrCore('cron-creator-to-user') as core:
+            msgs = await core.stormlist('cron.list')
+            self.stormIsInPrint('root                      root                      1ec949d8', msgs)
```

Comment:
Nit: you can add `whitespace=False` and ignore the repeating whitespace in the message.